### PR TITLE
Admin - soccer match

### PR DESCRIPTION
**Pull request by BaoKhanh99**
Related Tickets
- [#47143](https://edu-redmine.sun-asterisk.vn/issues/47143)
- [#47157](https://edu-redmine.sun-asterisk.vn/issues/47157)

What

- create soccer match
- update soccer match

How

- create and update soccer match by admin

Screenshot from 2022-03-25 10-04-02
Screenshot from 2022-03-25 10-05-14

### CODE REVIEW

**Comment by binhpt-1239** [comment]
ok

File: app/controllers/admin/soccer_matches_controller.rb [hunk 1]
```diff
@@ -1,6 +1,57 @@
...
     @pagy, @soccer_matches = pagy SoccerMatch.includes(:home_team, :guest_team)
                                              .newest
   end
+
+  def create
+    @soccer_match = SoccerMatch.new match_params
+    if @soccer_match.save
+
+      flash[:info] = t ".successful_create"
+      redirect_to new_admin_soccer_match_url
```
Comment (duypn-2245) [inline app/controllers/admin/soccer_matches_controller.rb:15]: chỗ này sao không dùng `new_admin_soccer_match_path` mà dùng `new_admin_soccer_match_url` vậy em ?

File: app/controllers/admin/soccer_matches_controller.rb [hunk 2]
```diff
@@ -1,6 +1,57 @@
...
+
   def index
     @pagy, @soccer_matches = pagy SoccerMatch.includes(:home_team, :guest_team)
                                              .newest
   end
+
+  def create
+    @soccer_match = SoccerMatch.new match_params
+    if @soccer_match.save
+
```
Comment (duypn-2245) [inline app/controllers/admin/soccer_matches_controller.rb:13]: remove line trống

File: app/controllers/admin/soccer_matches_controller.rb [hunk 3]
```diff
@@ -1,6 +1,57 @@
...
+  end
+
+  def edit; end
+
+  def update
+    if @soccer_match.update match_params
+      flash[:success] = t ".successful_update"
+      redirect_to admin_root_path
+    else
+      flash.now[:danger] = t ".successful_update"
```
Comment (duypn-2245) [inline app/controllers/admin/soccer_matches_controller.rb:33]: update failed mà mesages là succes à e ?

**Comment by binhpt-1239** [comment]
ok

**Review by duypn-2245 — APPROVED** [review]

**Pull request merged**